setup.py

```diff
@@ -20,8 +20,9 @@
     "imblearn>=0.0",
     "nltk>=3.2.4",
     "regex>=2019.03.12",
-    "lxml>=4.3.3"
-    "sentencepiece>=0.1.83"
+    "lxml>=4.4.1",
+    "sentencepiece>=0.1.83",
+    "cython>=0.29.14"
```

Comment:
What is cython used by?

Reply:
tbh I'm not completely positive...I do know that in addition to running setup.py, I had to additionally install cython in order for my code to run.

I'm guessing it's a dependency of sklearn that for some reason did not get installed with sklearn via setup.py. Not sure why, since I thought it handles dependencies as well, but I did hit this error when trying to install an editable version of finetune from scratch.